src/utils/validate.js

```diff
@@ -0,0 +1,16 @@
+// 숫자가 모두 양수가 아닌 경우
+export const isAllPositive = (numbers) => {
+  return numbers.every((num) => num > 0);
+};
+
+// 커스텀 문자열이 있는데 커스텀 형식에 맞지 않는 경우
+export const isValidCustomSeparator = (customString, customSeparators) => {
+  return !(customString && customSeparators.length === 0);
```

Comment:
저는 개인적으로 논리 연산이 얽혀있어 가독성이 약간 떨어지는 느낌을 받았습니다.
그래서 아래처럼 하는 것이 가독성이 나을 것 같다는 의견입니다:)
```suggestion
  return !customString || customSeparators.length > 0;
```

Reply:
말씀해주신 방법이 더 직관적인 것 같네요! 감사합니다👍👍👍